count_server/mix.exs

```diff
@@ -14,14 +14,33 @@ defmodule CountServer.Mixfile do
   end
 
   # Configuration for the OTP application.
-  #
+
```

Comment:
lol no, they came with phoenix new.  Deleting them is kind of like deleting prepackaged comments in your gemfile.  I got rid of them